### PR TITLE
use external KUBECONFIG

### PR DESCRIPTION
**Pull request by brybacki**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/cluster-deploy-prerequisites.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+# Wait until KubeVirt is ready
+count=0
+until _kubectl wait -n kubevirt kv kubevirt --for condition=Available --timeout 5m; do
+    ((count++)) && ((count == 5)) && echo "KubeVirt not ready in time" && exit 1
+    echo "Error waiting for KubeVirt to be Available, sleeping 1m and retrying"
+    sleep 1m
+done
+
+_kubectl wait -n cdi deployment/cdi-operator   --for=condition=Available --timeout=${KUBEVIRT_DEPLOYMENT_TIMEOUT}s
```
[the remark this answers is not in the document]
Comment (arnongilboa) [reply hack/cluster-deploy-prerequisites.sh:56]: what about the `kubevirt` namespace?

[the remark this answers is not in the document]
Comment (brybacki) [reply hack/cluster-deploy-prerequisites.sh:56]: Good question, but:
1. this is a part of script extracted from cluster-up, to make cluster up only start cluster, 
2. this is for cdi deploy that we manage with our yamls (I plan to remove that in near future and use CDI bundled with kubevirtCI), well, if anyone wants to do something like that and learn something about the plugin sources ;)
3. what about `kubevirt` namespace?

[the remark this answers is not in the document]
Comment (brybacki) [reply hack/cluster-deploy-prerequisites.sh:56]: https://github.com/kubevirt/kubevirt-velero-plugin/issues/37

[the remark this answers is not in the document]
Comment (akalenyu) [reply hack/cluster-deploy-prerequisites.sh:56]: oh right this is all upstream specific cluster setup, in that case it should be ok

**Comment by brybacki** [comment]
failures here are not related to the PR, general flakiness is under investigation here: https://github.com/kubevirt/kubevirt-velero-plugin/pull/36
/override pull-kvp-functional-test

**Pull request merged**